File: version_java/src/main/java/com/example/version_java/network/api/SheetApi.java
package com.example.version_java.network.api;

import com.example.version_java.network.response.CriminalResponse;

import org.jetbrains.annotations.NotNull;

import java.util.List;

import retrofit2.Call;
import retrofit2.http.GET;

public interface SheetApi {

    @GET("api/v1/d1skgjsjp83q0")
    @NotNull
    Call<List<CriminalResponse>> getSheetCriminals();
}

File: app/src/main/java/com/example/crimenotification/network/api/SheetApi.kt
package com.example.crimenotification.network.api

import com.example.crimenotification.data.model.Criminal
import retrofit2.Call
import retrofit2.http.GET

interface SheetApi {

    companion object {
        private const val URL = "api/v1/d1skgjsjp83q0"

    }

    @GET(URL)
    fun getSheetCriminals(): Call<List<Criminal>>

}